apps/voting/app/src/components/VotePanelContent.js

```diff
@@ -187,7 +187,7 @@ class VotePanelContent extends React.Component {
           </div>
           <div>
             <h2>
-              <Label>Quorum progress</Label>
+              <Label>Total yes votes</Label>
```

Comment:
I don't think `Total yes votes` really captures what is being shown (which is a % anyways)

![Screenshot_2019-04-15 Aragon](https://user-images.githubusercontent.com/9424721/56175379-d9d25f80-5fab-11e9-924a-73bc68becfca.png)

Users _have_ to know what min. quorum is to have intelligent conversations about a vote e.g. 
Alice: "Why didn't this vote pass even though it has more yes than no votes?" 
Bob: "The minimum quorum threshold wasn't reached." 

So I would actually change it to `Min. Quorum progress` since this is how we refer to it in onboarding and the user guide.

Reply:
I think quorum is a very technical term, and even I get really confused by it

Reply:
@luisivan then it could be explained, maybe with a tooltip next to the term. This is how it is explained in the User Guide:

> “Min. Quorum” is short for “Minimum Acceptance Quorum". Min. Quorum is the number for **what percent of the total outstanding supply of tokens must approve a proposal for the vote to be considered valid**. For example, if the Min. Quorum is set to 20%, then 20% of the outstanding token supply must vote to approve a proposal for the vote to be considered valid. If a vote does not make quorum, then it will fail, even if more tokens voted to approve the proposal than voted against it. For example, if the Min. Quorum is set to 20% and 10% of the outstanding token supply votes against the proposal but only 15% vote in support, then the proposal will fail because it has not reached the Min. Quorum threshold.

Reply:
@luisivan @john-light Last week I had a chat with @lkngtn about Aragon's quorum and how it differs from conventional quorum with the aim of adding clarity to the product on-boarding. He suggested that to rename 'Quorum' as ‘Minimum approval %’ because it’s more accurate. I personally like it because it might beeasier to understand by people that are confused by the definition of quorum in general, and because it's more precise as to how we use it.

I agree with John's idea, we should also offer further details/explanation by using a info popover or tooltip and should be consistent with the ones presented on the product on-boarding redesign. 

cc @bpierre @sohkai

Reply:
Retracted this change for now, proceeding to merge and will come back to the *quorum* term in the future